.github/workflows/check.yml

```
@@ -29,3 +29,21 @@ jobs:
         run: pyright
       - name: Run Ruff check
         run: ruff check
+
+  test-server:
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)